composer.json

```diff
@@ -42,6 +42,8 @@
             "src/"
         ]
     },
+    "minimum-stability": "dev",
```

Comment:
Again. Both settings works in tandem.

https://travis-ci.org/sebastianbergmann/php-code-coverage/jobs/87362951#L102-L149